dev/bots/test.dart

```diff
@@ -95,6 +95,29 @@ Future<Null> _verifyInternationalizations() async {
   print('Contents of $localizationsFile matches output of gen_localizations.dart script.');
 }
 
+Future<Null> _checkForTrailingSpaces() async {
+  if (!Platform.isWindows) {
+    final List<String> fileTypes = <String>['.dart', '.cxx', '.cpp', '.cc', '.h', '.java']
+      .map<String>((String type) => '--include=*$type').toList();
+    final List<String> excludedDirs = <String>['.pub-cache', 'cache', 'gen', 'build']
+      .map<String>((String type) => '--exclude-dir=$type').toList();
+    print('Checking for trailing whitespace in source files.');
+    await _runCommand('grep', fileTypes + excludedDirs +
```

Comment:
Shall we only check files that are changed in the PR? That way, the check could be faster and we don't have to remove all the trailing spaces in this PR.